compiler/rustc_target/src/spec/targets/arm64e_apple_tvos.rs

```diff
@@ -0,0 +1,24 @@
+use crate::spec::base::apple::{base, Arch, TargetAbi};
+use crate::spec::{FramePointer, Target, TargetOptions};
+
+pub(crate) fn target() -> Target {
+    let (opts, llvm_target, arch) = base("tvos", Arch::Arm64e, TargetAbi::Normal);
+    Target {
+        llvm_target,
+        metadata: crate::spec::TargetMetadata {
+            description: Some("ARM64e Apple tvOS".into()),
```

Comment:
The aarch64-apple-tvos target spec omits "Apple" from the description, as does the aarch64-apple-ios target spec.

Reply:
We have the same description for `arm64e_apple_darwin` & `arm64e_apple_ios`.

https://github.com/rust-lang/rust/blob/master/compiler/rustc_target/src/spec/targets/arm64e_apple_darwin.rs

https://github.com/rust-lang/rust/blob/master/compiler/rustc_target/src/spec/targets/arm64e_apple_ios.rs

I can remove it if it needs